frontend/components/service/comboForm.js

```diff
@@ -1,347 +1,372 @@
+        return serviceToAdd.length;
+    };
+    const SaveClose = () => {
+        handleSave(serviceToAdd);
+        handleClose();
+    };
+    const handleClose = () => {
+        setEditDialog(false);
+    };
+
+    return (
+        <Grid
+            container
+            direction="column"
+            justifyContent="center"
+            alignItems="center"
+        >
+            <Grid item={3}>
+                <IconButton onClick={extractAddServiceEdit}>
+                    <AddIcon/>
+                </IconButton>
+            </Grid>
+            <Dialog open={editDialog} fullWidth={true} scroll="body">
+                <DialogTitle>
+                    Edit Combo
+                    <IconButton
+                        aria-label="close"
+                        onClick={handleClose}
+                        size="medium"
+                        sx={{
+                            position: 'absolute',
+                            right: 10,
+                            top: 10,
+                        }}
+                    >
+                        <Close/>
+                    </IconButton>
+                </DialogTitle>
+                <DialogContent>
+                    <Stack spacing={2}>
+                        {!_isEmpty(serviceListAddable)  &&
+                            serviceListAddable.map((item) => (
+                                <ComboItem
+                                    key={item.id}
+                                    item={item}
+                                    handleServiceCheck={handleAddRemoveService}
+                                    removeService={removeService}
+                                    isEdit={true}
+                                    serviceCheckList={serviceToAdd}
+                                />
+                            ))}
+                    </Stack>
+                </DialogContent>
+                <DialogActions>
+                    <div className={styled.flexSpaceBetween}>
+                        <Button onClick={SaveClose}>Save </Button>
+                        <Button onClick={handleClose}>Cancel</Button>
+                    </div>
+                </DialogActions>
+            </Dialog>
+        </Grid>
+    );
 };
 
 const ComboForm = (props) => {
-  const {
-    openDialog,
-    handleCloseComboDialog,
-    serviceCheckList,
-    handleServiceCheck,
-    type,
-    setServiceCheckList,
-  } = props;
-  const [comboName, setComboName] = useState('');
-  const [comboPrice, setComboPrice] = useState(0);
-  const [comboDuration, setComboDuration] = useState(0);
-  const [editDialog, setEditDialog] = useState(false);
-  const isEdit = () => type == 'edit';
-  const handleSave = (serviceToAdd) => {
-    setServiceCheckList(serviceToAdd.concat(serviceCheckList));
-  };
-  useEffect(() => {
-    let name = '';
-    let price = 0;
-    let duration = 0;
-    for (let serviceItem of serviceCheckList) {
-      // console.log(serviceItem)
```

Comment:
clear this console.log

Reply:
The console.log is already removed, It was in the section of deleted compared to the previous.